index.html

```diff
@@ -0,0 +1,233 @@
+        </div>
+          
+
+
+    </div>
+
+    <div class="work-card">
+      <div class="work-img">
+        <img class="work-1-img" src="images/work/screenshot/snapshot-4.png" alt="Multi Post stories">
+      </div>
+      <h2 class="work-title">Multi-Post Stories</h2>
+      <div class="details-container">
+            <h5>CANOPY</h5>
+            <h6>Back End Dev</h6>
+            <h6>2015</h6>
+      </div>
+      <div class="project-desctiption">
+        <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="langs">
+            <button class="lang">html</button>
+            <button class="lang">css</button>
+            <button class="lang">JavaScript</button>  
+          </div>
+      </div>
+
+      <div class="work-button">
+        <button class="project-button">See Project</button>
+      </div>
+        
+
+
+    </div>
+
+
+  </section>
+
+  <section class="about">
+    <div class="header-container">
+      <h2 class="portfolio-header">About Myself</h2>
+    </div>
+
+
+
+  <div class="content-container">
+    <p class="content">I'm a software developer based in Lima, Peru. I can help you build a product, feature or website.
+    Take a look through some of my work and professional experience.
+    If you would are interested in my services, don't hesitate to contact me.
+  </div>
+
+
+
+  <div class="contact-container">
+    <h4 class="contact-title">LET'S CONNECT!</h4>
+  </div>
+
+  <div class="favicons-container">
+    <img class="favicon tw" src="images/header/social/Twitter-logo.png" alt="Twitter Icon">
+    <img class="favicon lk" src="images/header/social/Linkedin-logo.png" alt="Linkedin Icon">
+    <img class="favicon md" src="images/header/social/Medium-logo.png" alt="Medium Icon">
+    <img class="favicon gh" src="images/header/social/Github-logo.png" alt="GitHub Icon">
+    <img class="favicon ve" src="images/header/social/Vector-logo.png" alt="Vector Icon">
+  </div>
+
+  <div class="cont-resume-button">
+    <button class="resume-button">Get my resume</button>
+  </div>
+
+
+  <div class="languages">
+    <h3 class="languages-title">Languages</h3>
+  </div>
+
+    <ul class="languages-list">
+      <li class="web-lang">
+        <img class="lang-img" src="images/work/Icons/JS.png" alt="JavaScript">
+        <p> class="lang-title">JavaScript</p>
+      </li>
+      <li class="web-lang">
+        <img class="lang-img" src="images/work/Icons/HTML.png" alt="HTML">
+        <p class="lang-title">HTML</p>
+      </li>
+      <li class="web-lang">
+        <img class="lang-img" src="images/work/Icons/CSS.png" alt="CSS">
+        <p class="lang-title">CSS</p>
+      </li>
+    </ul>
+  
+<div class="frameworks-skills">
+  <div class="frameworks">
+    <h3 class="frameworks-title">Frameworks</h3>
+  </div>
+
+  <div class="skills">
+    <h3 class="skills-title">Skills</h3>
+  </div>
+</div>
```

Comment:
- [ ] Please, try to improve that part adding more styles to finish your amazing work 😉 

Figma design | Your design

![image](https://user-images.githubusercontent.com/60164239/133816395-955d07e0-ed1c-4a17-a7d1-7c36861dd0b8.png)

Reply:
- The issue here has not been properly resolved as the Languages, Framework and Skills section still needs styling.